clients/ui/frontend/src/components/ToastNotifications.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from 'react';
+import { AlertGroup } from '@patternfly/react-core';
+import ToastNotification from './ToastNotification';
+import { NotificationContext } from '~/app/context/NotificationContext';
+
+const ToastNotifications: React.FC = () => {
+  const [notifications] = useContext(NotificationContext);
```

Comment:
The current hook only gives access to notification actions. Should we expand it to also return the notifications list? This is why I'm using `useContext` here.